packages/start/config/index.d.ts

```diff
@@ -0,0 +1,14 @@
+import { InlineConfig } from "vite";
+
+type SolidStartInlineConfig = Omit<InlineConfig, "router"> & {
+    start?: {
+        ssr?: boolean,
+        extensions?: string[],
+        server?: any,
```

Comment:
if vinxi exposed the `NitroConfig` type this could be used instead of any

Reply:
I see.. well I can mention this to @nksaraf